docs/development/releasing.rst

```diff
@@ -299,29 +310,42 @@ packages that use the full bugfix/maintenance branch approach.)
    version number (with no ``v`` prefix). Once you are happy with the changes,
    click **Save**, then **Publish**.
 
+#. Send out release announcement emails to email lists in
+   `Getting Help with Astropy <https://www.astropy.org/help.html>`_. Use the
+   previous announcement as a template.
+
 .. _release-procedure-beta-rc:
 
 Modifications for a beta/release candidate release
 --------------------------------------------------
 
-   For major releases we do beta and/or release candidates to have a chance to
-   catch significant bugs before the true release. If the release you are
-   performing is this kind of pre-release, some of the above steps need to be
-   modified.
-
-   The primary modifications to the release procedure are:
-
-   * When entering the new version number, instead of just removing the
-     ``.dev``, enter "1.2b1" or "1.2rc1".  It is critical that you follow this
-     numbering scheme (``x.yb#`` or ``x.y.zrc#``), as it will ensure the release
-     is ordered "before" the main release by various automated tools, and also
-     tells PyPI that this is a "pre-release".
-   * Do *not* do the step of adding ``.dev`` in the "back to development" stage.
-     If an RC goes well, there's no need for a "dev" stage, as the same version
-     will be released with only minor doc updates, and strings like "x.yrcz.dev"
-     confuse some version number parsing tools.
-   * Do not do step #26 or later, as those are tasks for an actual release.
-
+For major releases, we do beta and/or release candidates to have a chance to
+catch significant bugs before the true release. If the release you are
+performing is this kind of pre-release, some of the above steps need to be
+modified.
+
+The primary modifications to the release procedure are:
+
+* When entering the new version number, instead of just removing the
+  ``.dev``, enter "1.2b1" or "1.2rc1".  It is critical that you follow this
+  numbering scheme (``x.yb#`` or ``x.y.zrc#``), as it will ensure the release
+  is ordered "before" the main release by various automated tools, and also
+  tells PyPI that this is a "pre-release."
+* Do *not* do the step of adding ``.dev`` in the "back to development" stage.
+  If an RC goes well, there is no need for a "dev" stage, as the same version
+  will be released with only minor doc updates, and strings like "x.yrcz.dev"
+  confuse some version number parsing tools.
+* Do not do steps in :ref:`post-release-procedure`.
+
+Once a release candidate is available, create a new Wiki page under
+`Astropy Project Wiki <https://github.com/astropy/astropy/wiki>`_ with the
+title "vX.Y RC testing" (replace "X.Y" with the release number) using the
+`wiki of a previous RC <https://github.com/astropy/astropy/wiki/v3.2-RC-testing>`_
+as a template.
+
+For releases that come after release candidates, the title of the changelog
```

Comment:
It's a bit awkward, but I think this goes in the "main" release instructions, right?  Because this section is for "beta/RC" but this note is for *non*-beta/RC releases

Reply:
I moved it up but you can tell me whether it is where you thought it is supposed to be. 🤞